rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -19,11 +19,6 @@
 # THE SOFTWARE.
 
 
-from amd.rocal import readers
```

Comment:
@LakshmiKumar23 - can you verify this change for rocAL?

Reply:
@kiritigowda These are required. Please make sure this file is reverted